containers/CompoundAndEarn/index.js

```diff
@@ -41,41 +41,25 @@ const CompoundAndEarn = () => {
   const { getLastSnowballInfo } = useAPIContext();
   const snowballInfoQuery = getLastSnowballInfo();
   const { userPools, userDeprecatedPools, loadedDeprecated,
-    sortedUserPools,setLoadedDeprecated,setSortedUserPools,
-    setUserPools } = useCompoundAndEarnContract();
+    sortedUserPools,setLoadedDeprecated,setSortedUserPools } = useCompoundAndEarnContract();
 
   const [search, setSearch] = useState('');
   const [type, setType] = useState('apy');
   const [userPool, setPool] = useState('all');
   const [lastSnowballInfo, setLastSnowballInfo] = useState([]);
-  const [lastSnowballModifiedInfo, setLastSnowballModifiedInfo] = useState([]);
-  const [filterDataByProtocol, setFilterDataByProtocol] = useState([]);
-  const [loadedSort, setLoadedSort] = useState(false);
-
-  //reset state when index opened
-  useEffect(()=>{
-    if(!loadedSort){
-      setSortedUserPools(false);
-      setLoadedSort(true);
-    }
-  },[loadedSort,setSortedUserPools])
+  const [firstSnowballInfo, setFirstSnowballInfo] = useState([]);
 
   useEffect(() => {
     if(userDeprecatedPools.length > 0 && !loadedDeprecated && sortedUserPools){
-      let newArray = [...userPools];
+      let newArray = [...lastSnowballInfo];
       userDeprecatedPools.forEach((pool) => {
-        //check if it's not duplicated
-        if(userPools.indexOf(
-          (element)=> element.address.toLowerCase() === pool.address.toLowerCase()
-        ) === -1){
-          newArray.push(pool);
-        }
+        newArray.unshift(pool);
```

Comment:
we don't want the deprecated pools to show on top, we want them to show on the ending of the user pools list